### ISSUE TITLE
 connect: no route to host

### ISSUE THREAD

**Issue by MASSKAgithub** [yassine/soxy-driver#11]

**Comment by MASSKAgithub** [comment]
Also when i do the curl testing command it outputs nothing, i created the network with user pass ip and port options, this isnt just working pls help

**Comment by yassine** [comment]
could you try this command instead:
docker run --rm -it --dns 8.8.8.8 --net soxy_network uzyexe/curl -v -s -k https://check.torproject.org/api/ip
notice the -k flag, I thinks the image needs a certificate update that's all

**Comment by MASSKAgithub** [comment]
Thanks, will try when i come back home

**Comment by MASSKAgithub** [comment]
So I've done and I'm getting this: 

* getaddrinfo(3) failed for check.torproject.org:443
* Couldn't resolve host 'check.torproject.org'
* Closing connection 0

The network was created with:
docker network create -d soxy-driver soxy_network  -o "soxy.proxyaddress"="iphere" -o "soxy.proxyport"="12324" -o "soxy.proxyuser"="\"user\"" -o "soxy.proxypassword"="\"pass\""

Help pls

**Comment by yassine** [comment]
This is a DNS resolution issue.
Are you running a (code) modified version of the code?

**Comment by MASSKAgithub** [comment]
No, I have no idea how to modify man, I'm a big noob.

But I think it's maybe my VPS provider ? I have an oracle AMD instance from the Free Tier.
I didn't open any port idk if its a problem.

And I've done only that:
```docker pull yassine/soxy-driver```
```docker run -d -v '/var/run/docker.sock':'/var/run/docker.sock' -v '/run/docker/plugins':'/run/docker/plugins' --net host --name soxy-driver --privileged yassine/soxy-driver```
```docker run --rm -it --dns 8.8.8.8 --net soxy_network uzyexe/curl -v -s -k https://check.torproject.org/api/ip```

Pls help

**Comment by MASSKAgithub** [comment]
OMG I FIXED

Ok so its the fault of my proxy provider, I just removed all rules of iptables and that worked:
``sudo iptables -F```
and then
``sudo iptables-save``

**Comment by MASSKAgithub** [comment]
OMGGGG

THat's working on my container


3 days that I've been looking for that thing

THank you so much that you made it im so happy 👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍

**Comment by MASSKAgithub** [comment]
NEW QUESTION ⚠️ 

Is it tunnelling UDP traffic?

**Comment by yassine** [comment]
TLDR: Regarding the kv store warning messages just ignore them.

The driver calls some code that is normally parts of a docker install. That code stores some values into a KV store for caching and maintaining state between reboots. That storage is not truly breaking for the soxy-driver as state maintenance is managed differently.

Regarding UDP tunneling it is theoretically feasible with the SOCKS5 protocol and redsocks supports it, but for now I do not have time to do it in the short term.
